bundles/org.openhab.binding.tr064/src/main/resources/channels.xml

```diff
@@ -65,7 +65,7 @@
 			serviceId="urn:X_AVM-DE_HostFilter-com:serviceId:X_AVM-DE_HostFilter1"/>
 		<getAction name="GetWANAccessByIP" argument="NewDisallow">
 			<parameter name="NewIPv4Address" thingParameter="wanBlockIPs"
-				pattern="^((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)(\.(?!$)|$)){4}$"/>
+				pattern="[1-9]\d{0,2}\.(\d{1,3}\.){2}[1-9]\d{0,2}(\s*#.*)*"/>
```

Comment:
This was recently fixed, please revert.

Reply:
The "fix" prevents adding comments. This is wrong and against the documentation.

Reply:
Sorry, i stared too much at the ip-address part and forgot about the optional comment.

Reply:
I guess your regex is more strict, so maybe we should find a solution that uses the more strict checking from your regex and the comments from mine.

Reply:
@lsiepel 

```
((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)(\.(?!([\s#]|$))|([\s#]|$))){4}(\s*#.*)*
```

fulfills

```
        assertTrue(pattern.matcher("192.168.0.1").matches());
        assertTrue(pattern.matcher("192.168.0.1 # foo").matches());
        assertTrue(pattern.matcher("192.168.0.1    # f00 # bar").matches());
        assertTrue(pattern.matcher("225.168.0.1").matches());
        assertTrue(pattern.matcher("1.0.0.1").matches());
        assertTrue(pattern.matcher("94.168.0.1").matches());
        assertFalse(pattern.matcher("322.168.0.1").matches());
        assertFalse(pattern.matcher("192.168.1").matches());
        assertFalse(pattern.matcher("192.168.0.1 foo").matches());
```

WDYT? The only invalid pattern I found that passes the regex is `0.168.1.2` because the single digit is not limited for the first group.

Reply:
Agree, don't have an immediate fix for that very specific case and it is an improvement anyway. Thanks!

Reply:
There is a solution, but that would nearly triple the length of the regex (1-254 for first group, a dot, 0-254 followed by dot (2 times), 1-254 for last group, then the comment regex). IMO that's overkill.

Reply:
Nah, I would opt your solution